.github/workflows/release.yml

```diff
@@ -61,11 +61,13 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-      - name: Extract branch name
-        id: extract_branch
-        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+      - name: verify release required
+        if: ${{steps.semantic-release.outputs.TAG_NAME == ''}}
+        uses: actions/github-script@v3
+        with:
+          script: |
+            core.setFailed('There are no relevant changes, so no new version is released.')
       - name: Update release
-        if: ${{steps.semantic-release.outputs.TAG_NAME != ''}}
```

Comment:
Should we remove this or let it be for extra security ?

Reply:
if the tag name is empty it will fail at line no 69. This would be a redundant check if we keep this.